dependencies.json

```diff
@@ -0,0 +1,616 @@
+{
+  "name": "dependencies",
+  "buildsystem": "simple",
+  "build-commands": ["mv vendor /app"],
```

Comment:
that's not how it works.

Reply:
remove lines 1 to 5, included, line 5 shuod just be `[`. And remove the last line.

Actually did you use https://github.com/flatpak/flatpak-builder-tools/tree/master/go-get ? If you didn't you should.

This should not be a `module`, but a list of sources.

Reply:
I tried both that tool and a few others, but from the docs:

![image](https://github.com/flathub/flathub/assets/28832235/59ea890f-731f-4839-8d88-e9204a9f6cb4)

The tool you mentioned doesn't support Go modules, plus its written in Python, so I used this instead which uses vendor: https://github.com/dennwc/flatpak-go-mod

Both of them seem to recommend just pasting the sources list into the main manifest though. I've checked other Go Flathub apps - they all do it like this, see for example:

- https://github.com/flathub/xyz.diamondb.gtkcord4
- https://github.com/flathub/io.github.noxworld_dev.OpenNox/blob/master/io.github.noxworld_dev.OpenNox.yml

Reply:
Nvm, figured it out - fixed in ee7a56fd23bf685107ae0fe6b048ba682190dc63